File: metadataviagens/Domain/Viaturas/TipoViaturaId.cs
using System;

namespace metadataviagens.Domain.Viaturas
{
    public class TipoViaturaId
    {
        public String id { get; private set; }

        public TipoViaturaId(String id)
        {
            this.id = id;
        }

        public String toString()
        {
            return this.id;
        }
    }
}

File: metadatarede/src/domain/tipoViaturaId.ts
import { Entity } from "../core/domain/Entity";
import { UniqueEntityID } from "../core/domain/UniqueEntityID";

export class TipoViaturaId extends Entity<any> {

    get id (): UniqueEntityID {
        return this._id;
    }

    public static create (id?: UniqueEntityID): TipoViaturaId {
        return new TipoViaturaId(id);
    }
}